provisioning/roles/solr-configuration/templates/solr_7.x_config-drupal8/schema.xml

```diff
@@ -49,7 +49,7 @@
     that avoids logging every request
 -->
 
-<schema name="drupal-8.3.8-solr-7.x" version="1.6">
+<schema name="drupal-4.1.0-solr-7.x-0" version="1.6">
```

Comment:
I'm not sure why this version number is used, but that's how it was downloaded from the site.